scripts/build-aarch64-linux-android

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/make -f
 
 TARGET := aarch64-linux-android
+BASE_TARGET := $(TARGET)
```

Comment:
I guess this serves the same purpose as the `VPX_ARCH` used previously? You can delete `VPX_ARCH` now that it's unused. :P

Reply:
Right